### DISCUSSION TITLE
Custom Model not loading instructions when loading through "@"

### DISCUSSION THREAD

**Discussion by DraculaVladimir** [open-webui/open-webui#2989] [General]
# Bug Report

## Description
Custom Model not loading instructions when loading through "@"

**Bug Summary:**
Custom Model not loading instructions when loading through "@"

**Steps to Reproduce:**
Load any model. Say "hi". Create a model called "Imagine" and 
![image](https://github.com/open-webui/open-webui/assets/52561202/9f582c63-c2a7-40ec-b6e7-86cc52dc0887)

**Expected Behavior:**
I expect the custom model to obey the instruction placed on "System Prompt"

**Actual Behavior:**
Model provides response as if the custom "System Prompt" did not exist.

## Environment
Open WebUI Version: v0.3.1
Ollama Version: 0.1.42 

- **Operating System: Ubuntu in WSL, on Windows 11

## Reproduction Details

**Confirmation:**

- [O] I have read and followed all the instructions provided in the README.md.
- [O] I am on the latest version of both Open WebUI and Ollama.
- [X] I have included the browser console logs.
- [O] I have included the Docker container logs.

## Logs and Screenshots
[docker.log](https://github.com/user-attachments/files/15776367/docker.log)

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Installation Method
docker run -d -p 3000:8080 --add-host=host.docker.internal:host-gateway -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main

**Comment by tjbck** [comment]
![image](https://github.com/open-webui/open-webui/assets/25473318/9503df83-139e-4e4a-91dd-be4286e97446)

![image](https://github.com/open-webui/open-webui/assets/25473318/e83ae878-17da-4227-a9e8-d2fc733b0847)

Seems to be a model specific issue here.

**Reply by DraculaVladimir** [reply]
I know there are multiple tickets to go through. 
I would appreciate another look at my ticket.
Load a normal LLAMA3 model or any other model available on [Ollama ](https://ollama.com/library)

Then type "@" and then select he custom system prompt
![image](https://github.com/open-webui/open-webui/assets/52561202/a0bb8c97-a3b6-4aae-a79d-fa7afb7ea42e)

in my case "imagine" model is meant to mimic everything I say.

**Reply by tjbck** [reply]
![image](https://github.com/open-webui/open-webui/assets/25473318/4f8eb7c6-f362-4c51-a178-bbb056c04cdb)

**Reply by tjbck** [reply]
Doesn't seem to be experiencing any issue with`@`either, I'd suggest you try changing your base model to something else.

**Reply by tjbck** [reply]
![image](https://github.com/open-webui/open-webui/assets/25473318/4d3a77d1-aeeb-4f36-b256-a35be31c4ad9)

![image](https://github.com/open-webui/open-webui/assets/25473318/2566afe6-6db1-4f08-9b22-d93c908c1dd7)
